packages/trader/src/App/Components/Elements/ContractCard/contract-card-item.jsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import React      from 'react';
+
+const ContractCardItem = ({
+    children,
+    content,
+    header,
+    is_crypto,
+    is_loss,
+    is_won,
+}) => {
+    return (
+        <div className='contract-card__item'>
+            <div className='contract-card__item__header'>
+                { header }
+            </div>
+            <div className={classNames('contract-card__item__body', {
```

Comment:
Maybe `contract-card-item` instead of `contract-card__item`? since we have `&__body`, it will be a `BEM` violation to have `contract-card__item__body`, its better to have it as `contract-card-item__body` to satisfy our `BEM` convention.

Reply:
sure, will rename it. it was written that way before